confluent-registry-schema/1.0/modules/ROOT/pages/confluent-schema-registry-connector-examples.adoc

```diff
@@ -0,0 +1,355 @@
+= Confluent Schema Registry 1.0 Examples - Mule 4
+
+This example shows how to publish and consume avro messages by integrating
+Confluent Schema Registry with Kafka Connector to reduce the size of the
+messages. The example contains two flows:
+
+. The first flow generates an avro message containing the schema and sends it
+to the Kafka topic. The *Transform Message* component creates an avro message
+with the schema and the *Replace AVRO schema with id* operation replaces the
+schema embedded in the message with an ID from Confluent Schema Registry.
+. The second flow reads the avro message from the Kafka topic and prints out the
+content of the deserialized avro message to the console. The
+*Replace id with AVRO schema* operation replaces the ID in the received message
+with the actual schema retrieved from Confluent Schema Registry.
+
+== Prerequisites
+
+You must have:
+
+* Java 8
+* Anypoint Studio 7.7.0 or later
+* Mule Runtime 4.2.x EE or later
+* DataWeave
+
+== Create the Producer Flow
+
+The Producer flow publishes a message to the configured Kafka topic. The
+following screenshot shows the Producer flow in Anypoint Studio:
+
+image::confluent-schema-registry-producer-flow.png[Confluent Schema Registry Connector Producer Flow in Studio]
+
+This flow involves configuring an HTTP *Listener* component, *Transform Message*
```

Comment:
Suggestion only: Make this a list

In this flow, you configure:
An HTTP Listener component
Transform Message component

...etc